package.json

```diff
@@ -21,7 +21,7 @@
     "checkchange": "node ./lib/cli.js check",
     "docs": "vuepress dev docs --host localhost",
     "docs:build": "vuepress build docs",
-    "prettier": "prettier --write --config ../../.prettierrc --ignore-path ../../.prettierignore 'src/**/*'",
+    "format": "prettier --write '**/*'",
```

Comment:
Having this command called `prettier` prevented running prettier directly with custom args. Switching to `format` makes that possible.